tests/components/FilterSearch.test.tsx

```diff
@@ -26,7 +26,7 @@ describe('search with section labels', () => {
   });
 
   it('renders the filter search bar, "Filter" label, and default placeholder text', () => {
-    render(<FilterSearch searchFields={searchFieldsProp} />);
+    render(<FilterSearch searchFields={searchFieldsProp} label='Filter' />);
```

Comment:
> We shouldn't need the `package-lock.json` or `test-site/package-lock.json` changes in this PR. The code doesn't add any new dependencies. Those files may have changed for a couple reasons: you weren't initially on the latest version of `main` or you have a different `npm`/Node version. Either way, you can un-stage the changes to these two files from your PR.

I believe the package-lock changed because in an earlier commit there was a react-router-dom dep was added (which has since been removed). Now that it's gone, if you rerun npm install in the base folder and test-site folder (or just unstage the changes like Tom suggested) the changes should go away.

Reply:
Could we just add a story? I think we probably don't need these "render checking" tests we keep adding

Reply:
We've kinda kept adding more jest tests that check rendering, which works but leads to pretty bloated test files